sql/threadpool_common.cc

```diff
@@ -214,7 +214,9 @@ int threadpool_add_connection(THD *thd)
 
   if (!setup_connection_thread_globals(thd))
   {
-    bool rc= login_connection(thd);
+    bool rc;
```

Comment:
I have not found anything in https://dev.mysql.com/doc/internals/en/general-development-guidelines.html. Given the typical function length in the core server code declaring variables at first use helps to keep sanity